content/sensu-go/5.8/guides/securing-sensu.md

```diff
@@ -37,6 +37,8 @@ etcd-peer-cert-file: "/path/to/your/peer/cert"
 etcd-peer-key-file: "/path/to/your/peer/key"
 etcd-peer-client-cert-auth: "true"
 etcd-peer-trusted-ca-file: "/path/to/your/peer/ca/file"
+etcd-cipher-suites
+ - your-cipher
```

Comment:
What should we include in this guide for this option?

Reply:
Tough call. This is an advanced option and I wouldn't necessarily want to prescribe something for this.